### PR TITLE
exploit php8 features + other cs improvements

### PR DESCRIPTION
**Pull request by garak**

### CODE REVIEW

File: src/Badge/Model/Package.php [hunk 1]
```diff
@@ -128,7 +125,7 @@ public static function parseStability(string $version): string
     {
         $version = \preg_replace('{#.+$}i', '', $version) ?? '';
 
-        if ('dev-' === \substr($version, 0, 4) || '-dev' === \substr($version, -4)) {
+        if (\str_starts_with($version, 'dev-') || '-dev' === \substr($version, -4)) {
```
Comment (AlessandroMinoccheri) [inline src/Badge/Model/Package.php:128]: Nice!

File: .php_cs.dist [hunk 2]
```diff
@@ -7,11 +7,18 @@ $finder = (new PhpCsFixer\Finder())
...
     ->setRules([
         '@Symfony' => true,
-        'array_syntax' => ['syntax' => 'short'],
+        '@Symfony:risky' => true,
+        '@PHP80Migration' => true,
+        '@PHP80Migration:risky' => true,
+        '@PHPUnit84Migration:risky' => true,
         'ordered_imports' => true,
-        'declare_strict_types' => false
+        'declare_strict_types' => false,
```
Comment (AlessandroMinoccheri) [inline .php_cs.dist:18]: Is too much for this PR to add `declare_strict_types`?

Comment (garak) [reply .php_cs.dist:18]: Maybe we can keep it for the next time?

**Review by AlessandroMinoccheri — APPROVED** [review]
LGTM!

**Review by JellyBellyDev — APPROVED** [review]
AWESOME!!! 👍

**Pull request merged**